src/segment-loader.js

```diff
@@ -1224,6 +1224,11 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     const Cue = window.WebKitDataCue || window.VTTCue;
     const value = {
+      bandwidth: segmentInfo.playlist.attributes['BANDWIDTH'],
```

Comment:
`attributes` may be `undefined` on the playlist object so you'll want to check for it before accessing the inner properties.

Also travis is failing because the linter prefers dot notation `playlist.attributes.BANDWIDTH`

Reply:
`attributes` being `undefined` was fixed in #1214. You do still need to change to dot notation though.